src/api/DashboardApiClient.ts

```diff
@@ -49,23 +49,11 @@ const onRedirectToLogin = () =>
   );
 
 export const DashboardApi = {

```

Comment:
Good for cleanup, remove the "V2" from the methods, we no longer need to disambiguate them